planning.md

```diff
@@ -117,6 +117,7 @@ You must consider the following:
 * How stored data will be migrated for stateful applications
 * How much downtime your application can tolerate during migration
 * How traffic will be redirected during migration
+* Restricted environment due to the networok policy (e.g. firewall), private image registry, and security restrictions
```

Comment:
@bryantson would you please share a little context?

Are you considering this statement to get the reader to consider environment restrictions in general that could be at play, or is it to call out specific differences seen in OCP 4 that customers may not have seen in OCP 3?

Note:  small typo in 'networok' to address before merging.

Reply:
@jwmatthews Also, I just forked and merged directly from my "master" to the main "master". Let me know if it is better to create a separate branch from my repo that was forked and do a PR from there instead.

Reply:
@bryantson 

Restricted environment is covered in the downstream MTC documentation:
- [Networking considerations](
https://docs.openshift.com/container-platform/4.5/migration/migrating_3_4/planning-migration-3-to-4.html#migration-preparing-networking)
- [Installing MTC in a restricted environment](https://docs.openshift.com/container-platform/4.5/migration/migrating_3_4/deploying-cam-3-4.html#installing-cam-operator_disconnected-3-4)

Was there something specific about restricted environment that is not covered by that documentation and that you wanted to add to the Best Practices Guide"

Reply:
@apinnick So, based on what you stated, I am guessing you don't want any duplicated information in this COP repo but only what is left out and missing?

Reply:
Almost. The downstream docs defines the procedures and requirements. This guide is more about recommendations and best practices -- things that might not be included in the downstream docs and might not be obvious to users who are planning to migrate from 3.x to 4.x. So this guide is a supplement to the downstream documentation.

If you happen to have a really great tip for migrating an application in a restricted environment that we haven't covered, this would be a good place to add it! But if you are reminding us to consider environment restrictions, that is probably a little too general for this guide. Thanks.

Reply:
Will take a heed. BTW, image URLs still need to fixed.

Reply:
> @jwmatthews Also, I just forked and merged directly from my "master" to the main "master". Let me know if it is better to create a separate branch from my repo that was forked and do a PR from there instead.

@bryantson in terms of creating PRs, the choice of which branch to use in your fork is a personal preference, as long as the PR is setup to merge into 'main' in this repo then we are good.   I tend to use separate branches per PR in my fork, helps me to manage different work streams a little easier.